apps/entity-service/src/entities/dto/site.dto.ts

```diff
@@ -85,5 +90,134 @@ export class SiteFullDto extends SiteLightDto {
   lightResource = false;
 
   @ApiProperty()
-  totalSiteReports: number;
+  siteReportsTotal: number;
```

Comment:
Please review the spreadsheet. In the usage notes for this column I requested that it be renamed to `totalSiteReports` to be consistent with projects. I'm going to skip over the rest of the DTO fields for now; please check what's on the spreadsheet against what you have here and I'll review again when that's done.